### PR TITLE
ANC Subsequent visits - Treatment Review activity

### PR DESCRIPTION
**Pull request by anvmn**
#247 
#248
#249
#250
#251
#252
#253

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-05-12 14-19

**Comment by anvmn** [comment]
## Referred to PMTCT:

Peek 2022-05-16 13-35:

**Comment by anvmn** [comment]
## Not referred to PMTCT:

Peek 2022-05-16 13-34

**Comment by anvmn** [comment]
@nedSaf please review

File: client/src/elm/Pages/Prenatal/Activity/Utils.elm [hunk 1]
```diff
@@ -357,12 +370,201 @@ expectNextStepsTask currentDate assembled task =
...
+
+latestMedicationTreatmentForHypertension : AssembledData -> Maybe Translate.TranslationId
+latestMedicationTreatmentForHypertension assembled =
+    let
+        treatmentOptions =
+            [ TreatmentMethyldopa2
+            , TreatmentMethyldopa3
+            , TreatmentMethyldopa4
+
+            -- @todo: extend with additional signs?
```
Comment (anvmn) [inline client/src/elm/Pages/Prenatal/Activity/Utils.elm:447]: @nedSaf this is a marker for future development

File: client/src/elm/Translate.elm [hunk 2]
```diff
@@ -4790,6 +4802,34 @@ translationSet trans =
...
+                    { english = "Patient Refused"
+                    , kinyarwanda = Nothing
+                    }
+
+                HIVTreatmentNoMedicineOther ->
+                    { english = "Other"
+                    , kinyarwanda = Nothing
+                    }
+
+                -- We do not require translation for reother signs.
```
Comment (nedSaf) [inline client/src/elm/Translate.elm]:
```suggestion
                -- We do not require translation for other signs.
```
?

**Review by nedSaf — APPROVED** [review]
Looks good, one minor typo, I suggested but not sure you meant `other`.
Fix and MWG

**Pull request merged**